deaf-and-hard-of-hearing/event-organizers/conference-best-practices.md

```diff
@@ -0,0 +1,141 @@
+# Best practices for an inclusive and welcoming conference
+
+If you're not familiar with the needs and preferences of deaf and hard of hearing (deaf/hoh) individuals, hosting a conference that is truly inclusive and accessible can feel overwhelming. This document will guide you to ensuring that deaf and hard of hearing attendees can fully participate in your event and have a great overall experience.  
+
+The first step of being more inclusive is understanding that deaf/hoh are not a one-size-fits-all group — we have different needs and preferences. Please refer to the [understanding the diverse needs of deaf and hard of hearing individuals](https://docs.google.com/document/d/1FQHbPP2asgp7NlCTXpjmhrARaXzCjN-DUogG4Ycf0xM/edit#heading=h.i8sn2yg7swqw) article to learn more about our community. You'll see why it is important to provide multiple options for deaf/hoh. If we can't participate, we will likely not return, reinforcing the vicious cycle of deaf/hoh individuals not participating in the community. We want to change that and need your help!
+
+To make the process as straightforward as possible, we've listed everything that would make a conference truly inclusive but also included alternatives if a particular accommodation is not feasible. If you have any questions or concerns, please get in touch with us at cncf-tag-contributor-strategy@lists.cncf.io. 
+
+**Note**: This document is open source. If we missed anything, please submit a pull request or issue so we can update it — thank you! 
+
+In this document, we'll cover:
+
+- [Website, registration, and CFP form](#website-registration-and-CFP-form)
+ [Pre-conference best practices](#pre-conference-best-practices)
+ [Day of best practices](#day-of-best-practices)
+    - [Reserved seating](#reserved-seating)
+    - [Sign language interpreters](#sign-language-interpreters)
+    - [Captions](#captions) 
+    - [Microphones for speakers and Q&A](#microphones-for-speakers-and-Q&A)
+    - [Be ADA compliant](#be-ada-compliant)
+    - [Meeting point area](#meeting-point-area)
+- [Conference networking opportunities](#conference-networking-opportunities)
+- [Ideas to be tested](#ideas-to-be-tested)
+
+## Website, registration, and CFP form {#website-registration-and-CFP-form}
+
+### Event website
+
+On the registration and call for papers (CFP) pages, state that accommodations will be made for speakers and attendees with disabilities. Clearly and prominently stating that will make us feel welcome and motivate us to participate. Don't bury that important information in a registration form we might never see. Here's some suggested copy:
+
+    Interested in attending or speaking but require special accommodations? Please inform us through the registration or CFP form about your specific needs, and we will make every effort to provide the necessary support. For accommodation-related questions, please reach out to [accommodations@yourevent.com](mailto:accommodations@yourevent.com).
```

Comment:
This should be indented. The preview seems to think differently.

Reply:
How about a quote block instead?

```suggestion
> Interested in attending or speaking but require special accommodations? Please inform us through the registration or CFP form about your specific needs, and we will make every effort to provide the necessary support. For accommodation-related questions, please reach out to [accommodations@yourevent.com](mailto:accommodations@yourevent.com).
```